src/main.rs

```diff
@@ -1,3 +1,7 @@
+// mod file_integrity;
+mod tool_settings;
+
 fn main() {
     println!("Hello, world!");
+    tool_settings::ToolsSettings::new();
```

Comment:
This looks good, but generally, most folks will also `use` any types they're using in the code.

i.e. 

```rust
use my_module::MyType;

mod my_module;
```

This allows you to refer to your types directly! :)